application/classes/Ushahidi/Repository/Contact.php

```diff
@@ -128,6 +135,21 @@ public function getByContact($contact, $type)
 		return $this->getEntity($this->selectOne(compact('contact', 'type')));
 	}
 
+    public function isInTargetedSurvey($contact_id)
+    {
+        $query = DB::select('targeted_survey_state.contact_id', 'targeted_survey_state.form_id')
+            ->from('targeted_survey_state')
+            ->where('contact_id', '=', $contact_id);
+
+        if($query->execute($this->db)->count() > 0)
+        {
+            Kohana::$log->add(Log::INFO, 'Contact is in a targeted survey: contact_id#'.print_r($contact_id, true));
```

Comment:
Are this still needed?

I actually really like the idea of INFO messages, I feel like any sufficiently large or small program should have INFO...it should really have LOG Levels implemented but INFO would be a good start because it makes debugging infinitely easier.